source/renderer/zettlr-body.js

```diff
@@ -657,39 +657,39 @@ class ZettlrBody {
       this._currentPopup = null
     }) // .makePersistent()
 
+    const searchForElement = document.getElementById('searchWhat')
+    const searchFor = () => searchForElement.value || ''
+    const replaceWithElement = document.getElementById('replaceWhat')
+    const replaceWith = () => replaceWithElement.value
     // If a regular expression was restored to the find popup, make sure to set
     // the respective class.
-    if (regexRE.test($('#searchWhat').val())) {
-      $('#searchWhat').addClass('regexp')
-      $('#replaceWhat').addClass('regexp')
+    if (regexRE.test(searchFor())) {
+      searchForElement.addClass('regexp')
+      replaceWithElement.addClass('regexp')
     }
 
     // Select the search input for convenience
-    $('#searchWhat').select()
+    searchForElement.select()
 
     // Another convenience: Already highlight all occurrences within the
     // document, if there is content in the find field.
-    if ($('#searchWhat').val() !== '') {
-      global.editorSearch.highlightOccurrences($('#searchWhat').val())
+    if (searchFor()) {
+      global.editorSearch.highlightOccurrences(searchFor())
     }
 
-    $('#searchWhat').on('keyup', (e) => {
-      this._findPopup.searchVal = $('#searchWhat').val()
-      if (regexRE.test($('#searchWhat').val())) {
-        $('#searchWhat').addClass('regexp')
-        $('#replaceWhat').addClass('regexp')
-      } else {
-        $('#searchWhat').removeClass('regexp')
-        $('#replaceWhat').removeClass('regexp')
-      }
+    searchForElement.addEventListener('keyup', (e) => {
+      this._findPopup.searchVal = searchFor()
+      const isInDocument = regexRE.test(searchFor())
```

Comment:
It's not `isInDocument`, but rather `isRegularExpression` (or, `isRegExp` for short) as it only checks if the search value is a regular expression or not